docs/training/apis-howto/api-estimator-ug.rst

```diff
@@ -2,6 +2,12 @@
  Estimator API
 ###############
 
+.. warning::
+
+   EstimatorTrial is deprecated and will be removed in a future version of Determined.  TensorFlow
+   has recommended Estimator users switch to Keras since the release of TensorFlow 2.0, and so
+   Determined recommends EstimatorTrial users switch to :class:`~determined.keras.TFKerasTrial`.
+
```

Comment:
EstimatorTrial is deprecated and will be removed in a future version. TensorFlow has advised Estimator users to switch to Keras since TensorFlow 2.0 was released. Consequently, we recommend users of EstimatorTrial to switch to the :class:`~determined.keras.TFKerasTrial` class.